.github/workflows/cypress-e2e-parallelization.yml

```diff
@@ -34,8 +34,22 @@ on:
         required: true
       registry_password:
         required: true
+      client_id :
+        required: true
+      client_secret:
+        required: true
+      jira_user: 
+        required: true
+      jira_token_test: 
+        required: true
+
 
 jobs:
+  get-version:
+    uses: ./.github/workflows/get-version.yml
+    with:
+      version_file: centreon/www/install/insertBaseConf.sql
```

Comment:
instead of calling get-version, could you add a new input to this workflow: `stability`